src/libopensc/card-openpgp.c

```diff
@@ -2312,10 +2311,10 @@ pgp_calculate_and_store_fingerprint(sc_card_t *card, time_t ctime,
 	if (key_info->algorithm != SC_OPENPGP_KEYALGO_RSA)
 		LOG_FUNC_RETURN(card->ctx, SC_ERROR_NOT_SUPPORTED);
 
-	if (modulus == NULL || exponent == NULL || mlen == 0 || elen == 0) {
-		sc_log(card->ctx, "Null data (modulus or exponent)");
-		LOG_FUNC_RETURN(card->ctx, SC_ERROR_INVALID_ARGUMENTS);
-	}
+	if (modulus  == NULL || mlen == 0 ||
+	    exponent == NULL || elen == 0)
+		LOG_TEST_RET(card->ctx, SC_ERROR_INVALID_ARGUMENTS,
+			     "Modulus or exponent missing");
```

Comment:
Deliberate indentation.

IMHO, it makes very much sense to 
* have elements subject to the same logic aligned to show the similarity.
  In my mind, having consecutive lines show up aligned just below the first one is way more readable than having them 
  * either indented one further TAB, which aligns them with the block controlled by the `if` condition, destroying the visual distinction between logical flow and representation
  * or by 2 TABs, which is even worse.
* have arguments to functions aligned with the opening brace, to show that they belong to that function.

BTW this type of alignment is also used in other parts of OpenSC's code (without me touching it).

Reply:
@frankmorgner I'd rather keep the indentation as is, especially as there is precedence for it in card-openpgp.c as wel as other files.
Changing it in this PR only would introduce yet another style of indentation.

How do we proceed from here? Please advise.

Reply:
We should not make assumptions about the developer's viewer. With your "Deliberate indentation" you're assuming a tab width of 8 characters, this is wrong.

If you really want to force a layout, you must use spaces only. Since this is not the case for OpenSC, please also use tabs for your code. Yes, we may have more bad indenting in the code base, but this does not justify introducing more such things.

Reply:
@frankmorgner 
I hope I got you right: "you do not mind aligned indenting, but it must be independent of TAB-length." Did I get that right?

Please find the new version, which only changes whitespace according to the following logic:
For wrapped-around statements, it keeps the same number of initial TABs as the starting statement, and adds up as many blanks as required.
This way, it
* keeps TABs wherever possible
* keeps indentation intact independent of the TAB length

Thanks for the advice / thanks in advance for pulling this PR into master
Peter

Reply:
@frankmorgner BTW I am will ing to adapt the openpgp card driver to the spacing used in the latest version of this PR.

Reply:
To be honest, I don't like it. When starting to indent things by hand, it always feels like the code is not simple and elegant enough to look good by itself.

Since we don't have a style guide (or let alone a consistent way of writing code), I don't care too much. But when I finally find time to implement a consistent formatting, I will definitely remove spaces that are followed by tabs and only use tabs for indenting.

Reply:
@frankmorgner Fine with me. 
* I save the time to re-align all wrap-around statements for the OpenPGP driver ;-)
* the PR does not contain spaces followed by tabs.

May I ask you pulling this PR
Peter

Reply:
tabs followed by spaces are equally ugly...